pandas/_libs/parsers.pyx

```diff
@@ -644,6 +646,7 @@ cdef class TextReader:
                     raise ValueError('Multiple files found in compressed '
                                      'zip file %s', str(zip_names))
             elif self.compression == 'xz':
+                if lzma is None: raise RuntimeError("lzma module not available.")
```

Comment:
Formatting: move the `raise` to its own line.

Reply:
Can you also note in the error message that the user may need to rebuild or reinstall Python with the proper dependencies to fix this.

Reply:
Do you want this message on the warning and on the RuntimeError?